controllers/etcd_controller.go

```diff
@@ -35,13 +35,10 @@ import (
 	"github.com/gardener/gardener/pkg/utils/imagevector"
 	kutil "github.com/gardener/gardener/pkg/utils/kubernetes"
 	gardenerretry "github.com/gardener/gardener/pkg/utils/retry"
-
 	"github.com/go-logr/logr"
 	appsv1 "k8s.io/api/apps/v1"
-
-	batchv1 "k8s.io/api/batch/v1beta1"
+	batchv1beta1 "k8s.io/api/batch/v1beta1"
```

Comment:
Can you remove the `batchv1beta1` changes? This import will entirely be removed by https://github.com/gardener/etcd-druid/pull/235 and thus will cause unnecessary merge conflicts.

Reply:
Done